### PR TITLE
Spondyr #3790

### PR DESCRIPTION
**Pull request by luancazarine**
Triggers
  - Spondyr Processed

Actions
  - Create Spondyr

### CODE REVIEW

File: components/spondyr/actions/create-spondyr/create-spondyr.mjs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import spondyr from "../../spondyr.app.mjs";
+
+export default {
+  key: "spondyr-create-spondyr",
+  name: "Create Spondyr",
+  version: "0.0.3",
```
Comment (alysonturing) [inline components/spondyr/actions/create-spondyr/create-spondyr.mjs:6]: please use version 0.0.1

File: components/spondyr/actions/create-spondyr/create-spondyr.mjs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    isGenerateOnly: {
+      type: "boolean",
+      label: "Is Generate Only",
+      description: "Set this flag to true to pause/defer delivery until a followup call is made via the Deliver API. This is useful for scenarios where you wish to view/preview the generated correspondence and validate things before allowing the correspondence to be delivered.",
+      optional: true,
+    },
+    data: {
+      type: "object",
+      label: "Data",
+      description: "It should contain your application's specific data, which should be in the same format as you defined for the `Transaction Type`",
```
Comment (alysonturing) [inline components/spondyr/actions/create-spondyr/create-spondyr.mjs:35]: is it possible to include a doc link to the expected format of this object? or an example of the expected object?

**Review by alysonturing — CHANGES_REQUESTED** [review]
Thanks for your contribution, I just noticed few points, please verify if the comments added are applicable.

File: components/spondyr/actions/create-spondyr/create-spondyr.mjs (hunk 2 above)
Comment (luancazarine) [reply components/spondyr/actions/create-spondyr/create-spondyr.mjs:35]: actually this can be any data that makes sense to the user

**Review by alysonturing — APPROVED** [review]
LGTM!

**Pull request merged**